algorithms/tendermint/verifier/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "tendermint-verifier"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# crates.io
+futures-util = "0.3.25"
+
+# ibc
+ibc = { path = "../../../ibc/modules", features = [] }
```

Comment:
do we need to specify `features = []`?

Reply:
Fixed here [0ef7d61](https://github.com/ComposableFi/centauri/pull/114/commits/0ef7d61f0a674f70cc61bccf1bb4872db8f218ef)